src/constants/FileDropzone.js

```diff
@@ -0,0 +1,27 @@
+export const baseStyle = {
```

Comment:
Don't have style here, but in a seperate file.
Put only logic and rendering components in js app files